scientific_library/tvb/simulator/plot/power_spectra_interactive.py

```diff
@@ -35,9 +35,9 @@
 ::
 
     #Load the demo data
-    import numpy
-    data = numpy.load("demos/demo_data_region_16s_2048Hz.npy")
```

Comment:
does this file still available?

Reply:
i think such files can be produced by some notebooks under "demo" folder

Reply:
Hi Julie, regarding that particular file, it is generated by a function called `generate_region_demo_data()` that is present in the notebook: tvb-root/tvb_documentation/demos/sandbox/pca_analyse_view_region.ipynb

Reply:
Hi Paula,
Thank you!

Should we not make this dataset available in the sandbox folder for the user, as it appears in several python files for usage case? Same for the file generate_region_demo_data.py (line 487) for the `LOG.error`?

Reply:
I suppose the generate_region_demo_data.py used to generate the same dataset (demo_data_region_16s_2048Hz.npy). I would suggest keeping only the function and run it automatically when the dataset is not found. 
Would it be better to make the `generate_region_demo_data` function available through the `tvb.simulator.lab `interface? @liadomide

Reply:
@JulieCB  I think this PR can be merged, and I can take care of making the dataset or the function available afterwards if that is ok from your side.